src/FakeItEasy/Expressions/ArgumentConstraints/EqualityArgumentConstraint.cs

```diff
@@ -37,7 +37,9 @@ public override string ToString()
 

```

Comment:
`ArgumentValueFormatter` already handles `null` (rendering it as `NULL` all the time, sadly not `<NULL>`) and strings (by default rendering either as `string.Empty` or `"thestring"`, but a custom formatter would override). I think we should defer to `ArgumentValueFormatter` all the time. Of course if we don't like `<NULL>` or `string.Empty`, we can adjust…

Reply:
Oops, I should have removed this. Will do.